src/assets/wise5/components/peerChat/peer-chat-question-bank/peer-chat-question-bank.component.ts

```diff
@@ -95,4 +121,8 @@ export class PeerChatQuestionBankComponent implements OnInit {
       })
     );
   }
+
+  private setQuestions(): void {
+    this.questions = this.displayedQuestionBankRules.flatMap((rule) => rule.questions);
+  }
```

Comment:
Would it be better to pass in displayedQuestionBankRules as a parameter?

Reply:
Yes, I made the change. PTAL